docs/grammar.md

```diff
@@ -51,6 +51,29 @@ Lark begins the parse with the rule 'start', unless specified otherwise in the o
 
 Names of rules are always in lowercase, while names of terminals are always in uppercase. This distinction has practical effects, for the shape of the generated parse-tree, and the automatic construction of the lexer (aka tokenizer, or scanner).
 
+## EBNF Expressions
+
+The EBNF expression in a Lark termminal definition is a sequence of items to be matched.
+Each item is one of:
+
+* `TERMINAL` - Another terminal, which cannot be defined in terms of this terminal.
+* `"string literal"` - Literal, to be matched as-is.
+* `"string literal"i` - Literal, to be matched case-insensitively.
+* `/regexp literal/` - Regular expression literal.  Can inclde flags.
```

Comment:
I think mentioned here where those flags can be placed is a good idea, not everyone is familar with that syntax:

```
* `/regexp literal/[imslux]` - Regular expression literal.  Can include the stdlib's `re` [flags `imslux`](https://docs.python.org/3/library/re.html#contents-of-module-re)
```

(also typo inclde -> include)

Reply:
Fixed in 200d6b5f004425174d2364cf701e964c3ca2e2a0.